runner_manager/models/backend.py

```diff
@@ -184,6 +187,8 @@ def configure_instance(self, runner: Runner) -> AwsInstance:
                 Tags=tags,
             ),
         ]
+        iam_instance_profile: IamInstanceProfileTypeDef = IamInstanceProfileTypeDef(Name=self.iam_instance_profile_name)
```

Comment:
Not sure if this needs to be typed as it can never not be `IamInstanceProfileTypeDef`

Reply:
Yeah up to you, I have no personal preference and as long as the editor understand I'm happy. (which I think it does if you remove the type).